src/Couchbase.Lite.Tests.Shared/ReplicationTest.cs

```diff
@@ -1339,10 +1339,10 @@ public void TestNonBlockingDatabaseOperationConflictResolver()
                 if (resolveCnt == 0) {
                     using (var d = Db.GetDocument("doc1"))
                     using (var doc = d.ToMutable()) {
-                        d.GetString("name").Should().Be("Tiger");
+                        d.GetString("name").Should().Be("Cat");
                         doc.SetString("name", "Cougar");
                         Db.Save(doc);
-                        d.GetString("name").Should().Be("Cougar", "Because database save operation was not blocked");
+                        doc.GetString("name").Should().Be("Cougar", "Because database save operation was not blocked");
```

Comment:
It's good to fix the exception, but this is not the only thing that needs to happen.  What happens if this fails?  How will the test know?  If an exception is thrown inside a conflict resolver then the exception is swallowed, logged, and the resolution is aborted. This test still passes in this case, as it was passing before even though this logic was wrong.  That needs to be corrected.